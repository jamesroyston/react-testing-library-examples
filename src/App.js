import React from 'react';
import './App.css';
import Header from './components/Header/Header';

function App() {
  return (
    <div className="App">
      <header className="App-header">
        <Header text="Hello!" />
      </header>
    </div>
  );
}

export default App;
