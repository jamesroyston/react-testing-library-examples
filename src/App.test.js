import React from 'react';
// import ReactDOM from 'react-dom';
import { render, cleanup } from '@testing-library/react';
import App from './App';

it.skip('renders without crashing', () => {
  const {container} = render(<App />);
  expect(container).toMatchSnapshot();
});
