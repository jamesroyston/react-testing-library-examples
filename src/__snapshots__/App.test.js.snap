// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div>
  <div
    class="App"
  >
    <header
      class="App-header"
    >
      <header>
        <h1
          class="fancy-h1"
          data-testid="h1"
        >
          Hello!
        </h1>
      </header>
    </header>
  </div>
</div>
`;
